pkg/rke2/control_plane.go

```diff
@@ -109,6 +120,22 @@ func (c *ControlPlane) AsOwnerReference() *metav1.OwnerReference {
 	}
 }
 
+// EtcdImageData returns the etcd image data embedded in the ClusterConfiguration or empty strings if none are defined.
+func (c *ControlPlane) EtcdImageData() (string, string) {
+	return "", ""
```

Comment:
Do we need some logic here to read from `ClusterConfiguration`?

Reply:
I would say no, we don't need this part because it is mostly related to external ETCD databases which is not supported by RKE2